app/api/views/apple.py

```diff
@@ -8,14 +8,17 @@
 from flask_cors import cross_origin
 
 from app.api.base import api_bp, require_api_auth
-from app.config import APPLE_API_SECRET
+from app.config import APPLE_API_SECRET, MACAPP_APPLE_API_SECRET
 from app.extensions import db
 from app.log import LOG
 from app.models import PlanEnum, AppleSubscription
 
 _MONTHLY_PRODUCT_ID = "io.simplelogin.ios_app.subscription.premium.monthly"
 _YEARLY_PRODUCT_ID = "io.simplelogin.ios_app.subscription.premium.yearly"
 
+_MACAPP_MONTHLY_PRODUCT_ID = "io.simplelogin.macapp.subscription.premium.monthly"
+_MACAPP_PRODUCT_ID = "io.simplelogin.macapp.subscription.premium.yearly"
```

Comment:
@nguyenkims : `_MACAPP_PRODUCT_ID` should be `_MACAPP_YEARLY_PRODUCT_ID`

Reply:
ah you're right! Let me fix it